content/docs/get-started/index.md

```diff
@@ -165,3 +162,363 @@ requirements:
+  return the class probabilities, instead of the default `predict`.
+- The `--import`/`-i` flag tells MLEM to import the data on the fly.
+- The `--import-type` / `--it` flag, helps MLEM understand the data format.
+  Here, it's `pandas[csv]` - a csv file that should be read with Pandas. For
+  that to work, your data should be in a format that is supported by
+  [MLEM import](/doc/user-guide/importing). You can learn more about specifying
+  these arguments on `mlem apply` page.
+
+Alternatively, you could save the dataset itself
+[using MLEM](/doc/user-guide/data) to use `mlem apply` on it.
+
+</details>
+
+</tab>
+</toggle>
+
+We saw that MLEM provides a consistent and friendly way for you to work with
+models both via API and CLI. However, MLEM **really** shines when you need to
+package and deploy your models, either as part of an application or locally for
+testing.
+
+## Deploying and Serving models
+
+MLEM can serve a model for you using different server implementations, for
+example FastAPI or RabbitMQ. Here we'll check out how it works with FastAPI
+since serving models via a REST API is a very common use case.
+
+## Local model serving
+
+First thing first, let's run a model server locally on our machine. To launch a
+local FastAPI model server, simply run:
+
+```cli
+$ mlem serve fastapi --model models/rf
+⏳️ Loading model from models/rf.mlem
+Starting fastapi server...
+🖇️  Adding route for /predict
+🖇️  Adding route for /predict_proba
+🖇️  Adding route for /sklearn_predict
+🖇️  Adding route for /sklearn_predict_proba
+Checkout openapi docs at <http://0.0.0.0:8080/docs>
+INFO:     Started server process [16696]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
+```
+
+The server is now running and listening on `http://0.0.0.0:8080` for HTTP
+requests.
+
+Servers automatically create endpoints from model methods using the
+`sample_data` argument provided to [`mlem.api.save`](/doc/api-reference/save).
+
+<admon type="tip">
+
+Servers automatically create endpoints from model methods using the
+`sample_data` argument provided to [`mlem.api.save`](/doc/api-reference/save).
+
+</admon>
+
+#### Making requests
+
+While the model server is running, you can use your browser to open the Swagger
+UI (OpenAPI) at [http://localhost:8080/docs](http://localhost:8080/docs) and
+check out OpenAPI spec and query examples.
+
+<admon type="info">
+
+MLEM can also generate client-side code to query/infer the model server. Learn
+more about this in
+[Serving User Guide](/doc/user-guide/serving#making-requests).
+
+</admon>
+
+## Deploying models to production
+
+Now, let's take model serving a step further and use production worthy
+deployment technologies. MLEM lets you easily package and deploy your models
+using a variety of platforms like [Docker](/doc/user-guide/deploying/docker),
+[Heroku](/doc/user-guide/deploying/heroku),
```

Comment:
...Is this about deploying, packaging, building? 😕

Reply:
U: I've clarified the structure in https://github.com/iterative/mlem.ai/pull/265.